docs/DeveloperGuide.adoc

```diff
@@ -731,64 +731,70 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
-|`* * *` |user |create a password to hide all the protected field |only I can access the protected field
-
-|`* * *` |user with sensitive data|log in to access the addressbook |only I can access the addressbook
-
-|`* * *` |user |store my addressbook in the cloud |I can access my addressbook from any device
+|`* * *` |user |log in to access the addressbook |only I can access the addressbook
 
 |`* * *` |user |list the names only in my addresbook |I don't have to see all the details of every person I am not interested in
 
-|`* * *` |user |see all the names in my contact without the details |I can view a compact list of names on my addressbook
-
 |`* * *` |user |search for users by other fields (e.g. email/address/contact number) |I can quickly access contact by other identifiable attributes
 
-|`* * *` |user |add an event to the addressbook |I can easily save appointments to the addressbook's calendar
+|`* * *` |user |open a save file in another directory |I can have multiple addressbooks and share with other users
 
-|`* * *` |user |execute commands interactively |I don't need to remember all the commands
+|`* * *` |user |change the color scheme/UI elements of the addressbook |I can select a color/font that suits my preference
 
-|`* * *` |user |open a save file in other directory |I can have multiple addressbooks and share with other users
+|`* * *` |user |export and import my contact details |I can share the contact details
 
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |share a contact card (that is exported) with another user |I can easily share with a click of a button and avoid manual sharing
+
+|`* * *` |user |add my contact's birthday details |I can keep track of my contact's birthdays
 
-|`* *` |user |be reminded of my contact's birthday |I can remember their birthdays easily
+|`* * *` |user |be reminded of my contact's birthday |I can remember their birthdays easily
 
-|`* *` |user |view my contact's social media profiles |I can easily connect with them if I have not already done so
+|`* * *` |user |add social media links for me and my contacts |We can connect socially
 
-|`* *` |user |execute commands with my voice |I can search for contacts without having to touch the keyboard
+|`* * *` |user |view my contact's social media profiles link |I can easily connect with them if I have not already done so
 
-|`* *` |user |interact with the GUI in natural language |I do not have to remember the commands
+|`* * *` |user |use alias for shorthand for commands |I don't have to type all the commands every time I use it
 
-|`* *` |user |edit their details without inputting a long String |I can conveniently edit a person without having to include tags every time
+|`* * *` |user |auto-complete the commands |I can be efficient and not worry about entering wrong commands
```

Comment:
many of these should have a lower priority. Not sure if you realise that 3 star is the highest priority. looks worse than the previous version